output-core/src/CoreOutputTemplateConverterStrategy.ts

```diff
@@ -12,20 +12,22 @@ export class CoreOutputTemplateConverterStrategy extends OutputTemplateConverter
   OutputTemplateConverterStrategyConfig
 > {
   responseClass = CoreResponse;
-  platformName = 'Core';
 
```

Comment:
`platformName` is abstract in `OutputTemplateConverterStrategy` and therefore required in this strategy.